tests/unit/test_UIARemoteOps/test_highLevel/test_bool.py

```diff
@@ -0,0 +1,91 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2023 NV Access Limited
+
+"""
+High-level UIA remote ops Unit tests for setting and comparing booleans.
+"""
+
+from unittest import TestCase
+from UIAHandler._remoteOps import operation
+from UIAHandler._remoteOps import remoteAPI
+
+
+class Test_bool(TestCase):
+
+	def test_bool_false(self):
```

Comment:
Since "Test_bool" is in the test suite name these could probably be simplified. Same in other files

```suggestion
	def test_false(self):
```